orttraining/orttraining/training_ops/cuda/optimizer/gradient_control.cc

```diff
@@ -79,7 +79,7 @@ Status InPlaceAccumulator<T, T_GRAD>::ComputeInternal(OpKernelContext* ctx) cons
       reinterpret_cast<const CudaT_GRAD*>(right_addee_buffer.template Data<T_GRAD>()),
       reinterpret_cast<CudaT*>(accumulation_output.template MutableData<T>()),
       right_addee_buffer.Shape().Size());
-
+  
```

Comment:
remove？

Reply:
Ok.

---
In reply to: [538709827](https://github.com/microsoft/onnxruntime/pull/5815#discussion_r538709827) [](ancestors = 538709827)